CHANGES.md

```diff
@@ -5,6 +5,12 @@
   Import '(<>)' from Semigroup, not Monoid.
 * [#80](https://github.com/serokell/universum/issues/80):
   Rename `Container` to `ToList`, `NontrivialContainer` to `Container`.
+  Add `NonTrivialContainer` type.
```

Comment:
You don't `Add` you `Keep old name as type alias`. Also not `NonTrivialContainer`, it's `NontrivialContainer`.